### ISSUE TITLE
TensorBoard in PyTorch Lightning

### ISSUE THREAD

**Issue by souadkhalifeh** [OHF-Voice/piper1-gpl#146]
Hello everyone,

While fine-tuning Piper, I’m monitoring training using TensorBoard with PyTorch Lightning 

In TensorBoard 2.20.0:
- I do see val_loss
- I also see loss_d and loss_g
- But no other metrics appear
- And the Graphs tab does not appear at all

What’s confusing is: 
- loss_d and loss_g are shown without any explanation of how they relate to the overall training objective
- Aside from val_loss, no additional validation or training metrics are logged
- Even though training and validation are running normally, TensorBoard feels very minimal compared to standard Lightning setups

My questions:
- What do loss_d and loss_g represent internally in Piper training?
- Is it expected that Piper logs only these losses (plus val_loss) and nothing else?
- Why might the TensorBoard Graphs tab be missing entirely in this setup?
- Is there a recommended way to enable richer metric or graph logging when training Piper with Lightning?

Thanks in advance.

<img width="1900" height="888" alt="Image" src="https://github.com/user-attachments/assets/44ff7220-566d-40a7-b27a-dcceb7abdf19" />
